eng/Versions.props

```diff
@@ -4,22 +4,13 @@
     <VersionPrefix>1.0.0</VersionPrefix>
     <PreReleaseVersionLabel>prerelease</PreReleaseVersionLabel>
   </PropertyGroup>
-  <!-- Package names -->
-  <PropertyGroup>
```

Comment:
You can leave this if you want.  I added support for creating the property group in my latest arcade PR.

Reply:
Currently the default xmlns https://github.com/dotnet/corefx/pull/33014/files#diff-8b8f08ffbf7b863fb3700c1718eeb4cbR2 will cause the updates not to happen here.  Fixed in https://github.com/dotnet/arcade/pull/1140